src/mgmt/srv_util.c

```diff
@@ -80,3 +80,17 @@ ds_mgmt_group_update(crt_group_mod_op_t op, struct server_entry *servers,
 		d_rank_list_free(ranks);
 	return rc;
 }
+
+void
+ds_mgmt_kill_rank(bool force)
```

Comment:
It looks like there are ways a kill call can fail. Not sure how plausibly in this case, but it could be worth returning a status code.